src/cloud/env/variables-build.md

```diff
@@ -201,6 +201,19 @@ stage:
     SKIP_SCD: false
 ```
 
+### `SKIP_COMPOSER_DUMP_AUTOLOAD`
+
+Skip executing of `composer dump-autoload` command.
```

Comment:
```suggestion
Skip execution of `composer dump-autoload` command.
```

Reply:
Suggest: Avoid using terms like execute and terminate:

 Do not run the `composer dump-autoload` command.

Why do you need to set this variable?  What does it accomplish?

Reply:
@meker12 Executing `composer dump-autoload` command causes an error when running some Magento commands such as `bin/magento deploy:mode:set production` or `bin/magento -n setup:upgrade`. These commands can be run on writable systems, for example in the magento-cloud-docker `test` container. So we added a possibility to skip it.
This is a core issue https://github.com/magento/magento2/issues/23251, but I don't know when it will be fixed.